pupil-spa/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
+  apiURL: 'API_URL',
```

Comment:
i couldnt get this to work, but then realised i had overwritten the entry with undefined.  I had to switch it back so it would pick up the real value.  This could be problematic in development, but we should only really be updating this file during the docker image build, outside of local development.  If it proves problematic then we should have a separate 'template' file for environment.prod.ts and update the target.